packages/store/src/patterns/rankStore.js

```diff
@@ -0,0 +1,119 @@
+// @ts-check
```

Comment:
Do you now use this file?

Reply:
As best I can discern, the file `rankStore.js` is mentioned no place in AgoricSDK other than itself.  Should it be removed?

Reply:
@erights I'm going to remove this file from the PR unless you object soonish.